checkbox-snap/series_uc16/launchers/shell-wrapper

```diff
@@ -4,4 +4,4 @@
 
 echo "$SNAP_NAME runtime shell, type 'exit' to quit the session"
 
-exec bash
+exec bash --rcfile <(cat ~/.bashrc; echo 'PS1="(checkbox-shell) $PS1"')
```

Comment:
The reading of `~/.bashrc ` looks new. It is needed for anything?

Reply:
As I added in the issue:

Python virtual environment is not creating another bash session, so they can modify the prompt (PS1), and works fine.
Since our script is creating a new bash session, it is calling .bashrc and PS1 gets reset.

A way to go over this is adding a new line to your .bashrc with the new PS1 update and parsing it as an argument to bash:
```
# shell-wrapper
- exec bash
+ exec bash --rcfile <(cat ~/.bashrc; echo 'PS1="(checkbox-shell) $PS1"')
```
So, if I don't have the  `--rcfile` as an argument, your bash rc will execute and overwrite the $PS1 in the new bash session.

Reply:
Another option that I tested was to directly ignore your `.bashrc` and have a custom prompt for the checkbox bash, but that will override your current PS1

```
# Customize PS1 for session
PS1="(checkbox) \e[0;32m\]\u\e[0;1m\]@\e[0;33m\]\h \e[0;34m\]\w \e[0;1m\]$ \e[m"
export PS1

exec bash --norc
```

Reply:
The more I think about it, the more I see how much off I was :)
Indeed, this looks like the right way forward!
Thanks for the great explanation!